proj/src/main/webapp/script.js

```diff
@@ -344,12 +353,10 @@ function makeChatPreview(name, messageObj, tag, chatId) {
 }
 
 function populateSidebar() {
-    // Initialize auth object
-    const auth = firebase.auth();
-
+    // const auth = firebase.auth();
     // const iud = auth.currentUser.uid;
     const iud = "testuserwithdict";
```

Comment:
Should this still be the test user or the real user?

Reply:
it's still the test user, nick said he was making a PR to change that i believe

Reply:
I am indeed.